astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -15,14 +15,8 @@
 import pytest
 import numpy as np
 
-from astropy.table import Table, Column, QTable, NdarrayMixin
```

Comment:
In this file I refactored the serialization testing out to a new table module `test_serialize`. The goal is to unify such testing for FITS, HDF5 and ECSV. However this proved a little tricky on my first attempt so the refactor only  currently applies to ECSV. In the context of this PR that should be OK?